Doc/includes/email-unpack.py

```diff
@@ -43,7 +43,7 @@ def main():
             if not ext:
                 # Use a generic bag-of-bits extension
                 ext = '.bin'
-            filename = 'part-%03d%s' % (counter, ext)
+            filename = f'part-{counter:03}{ext}'
```

Comment:
The 'd' should still be included, to explicitly mark it as a decimal. It may not be 100% necessary in this particular case, but I think we should still include it since the original example did. 
```suggestion
            filename = f'part-{counter:03d}{ext}'
```